package.json

```diff
@@ -11,13 +11,16 @@
     "postversion": "git push && git push --tags",
     "test": "jest",
     "lint": "tslint --project .",
+    "prestart": "npx sequelize db:migrate",
```

Comment:
if multiple instances get restarted at the same time we may have multiple concurrent migrations running, please check whether there is some kind of locking mechanism

Reply:
apparently answer is no:

https://github.com/sequelize/umzug/issues/59

https://github.com/sequelize/umzug/issues/144

Reply:
somebody suggested to use [this](https://github.com/SGrondin/umzug-sync), please check the WARNING at the bottom

Reply:
According to the [warning](https://github.com/SGrondin/umzug-sync#warning), the race conditions can still occur if the execution time for the migrations exceeds the timeout (default to 15 s).
The execution of the three current migrations in the my local dockerized environment takes less than 100 ms; of course the execution in the production environment will be affected by some factors that locally have no role (e.g. the speed connection), but to me even the default value looks already good enough, considering that the execution time for each migration can be counted in tens of milliseconds.